main.rb

```diff
@@ -0,0 +1,54 @@
+require_relative 'app'
+class Main
+  def initialize
+    @app = App.new
+  end
+
+  # rubocop:disable Metrics/CyclomaticComplexity
```

Comment:
- [ ] Kindly fix this Linter error. Do not disable them. Linters help to detect and flag programming errors.